### PR TITLE
Bug/v1 140 | Fixed long skill names going out of the border & added hover to showing full skill name

### PR DESCRIPTION
**Pull request by PechurinaLena**

### CODE REVIEW

File: frontend/src/pages/Profile/UserSkills/SkillInfo.tsx [hunk 1]
```diff
@@ -27,23 +28,35 @@ const CourseMaterialInfo: React.FC<Props> = ({
...
-        </SkillInfoStage>
-      </div>
-      <SkillProgress>
-        <ProgressBar value={progress} size={SIZE.small} />
-      </SkillProgress>
-    </SkillInfoFlex>
-  </SkillInfo>
-);
+}) => {
+  const [isShown, setIsShown] = useState(false);
```
Comment (irina-sinitsyna) [inline frontend/src/pages/Profile/UserSkills/SkillInfo.tsx:32]: Please move to SkillUserContainer and remove return statement, here shouldn't be any logic

File: frontend/src/pages/Profile/UserSkills/SkillInfo.tsx [hunk 2]
```diff
@@ -27,23 +28,35 @@ const CourseMaterialInfo: React.FC<Props> = ({
...
+
+  return (
+    <SkillInfo completed={stages === stagesCompleted}>
+      <SkillInfoFlex>
+        <ImageWrapper>
+          <UserAvatar size={SIZE.small} avatar={imageUrl} />
+        </ImageWrapper>
+        <div style={{ position: 'relative' }}>
```
Comment (irina-sinitsyna) [inline frontend/src/pages/Profile/UserSkills/SkillInfo.tsx:40]: Please use styled.ts for styling components

File: frontend/src/pages/Profile/UserSkills/SkillInfo.tsx [hunk 3]
```diff
@@ -27,23 +28,35 @@ const CourseMaterialInfo: React.FC<Props> = ({
...
+          <div style={{ width: '70px' }}>
```
Comment (irina-sinitsyna) [inline frontend/src/pages/Profile/UserSkills/SkillInfo.tsx:41]: Like the previous one

File: frontend/src/pages/Profile/UserSkills/SkillInfo.tsx [hunk 4]
```diff
@@ -27,23 +28,35 @@ const CourseMaterialInfo: React.FC<Props> = ({
...
+            <SkillInfoText
+              onMouseEnter={() => setIsShown(true)}
```
Comment (irina-sinitsyna) [inline frontend/src/pages/Profile/UserSkills/SkillInfo.tsx:43]: Maybe you should create separate functions (in SkillsInfoContainer) and reuse them here like this:
`onMouseEnter={showSkillInfo}`

File: frontend/src/pages/Profile/UserSkills/SkillInfo.tsx [hunk 5]
```diff
@@ -27,23 +28,35 @@ const CourseMaterialInfo: React.FC<Props> = ({
...
+              onMouseLeave={() => setIsShown(false)}
```
Comment (irina-sinitsyna) [inline frontend/src/pages/Profile/UserSkills/SkillInfo.tsx:44]: Like the previous one

**Review by irina-sinitsyna — CHANGES_REQUESTED** [review]

**Review by irina-sinitsyna — APPROVED** [review]

**Review by Julia-Maliutsina — APPROVED** [review]

**Pull request merged**